usaspending_api/awards/models.py

```diff
@@ -84,6 +84,17 @@ class Meta:
         db_table = 'financial_accounts_by_awards_transaction_obligations'
 
 
+class AwardManager(models.Manager):
+    def get_queryset(self):
+        q_kwargs = {
+            "type": "U",
+            "total_obligation__isnull": True,
```

Comment:
Just curious--where did the criteria to include null total_obligation, date_signed, and recipient come from?